poky.xml

```diff
@@ -3,13 +3,16 @@
 <!-- SPDX-License-Identifier: MIT -->
 <manifest>
 
-  <remote fetch="ssh://git@github.com" name="github"/>
-  <remote fetch="http://git.yoctoproject.org" name="yocto"/>
+  <remote fetch="https://github.com" name="github"/>
+  <remote fetch="https://git.yoctoproject.org" name="yocto"/>
 
-  <default revision="warrior" sync-j="4"/>
+  <default revision="master" sync-j="4"/>
 
-  <project name="armmbed/meta-mbl" path="layers/meta-mbl" remote="github" revision="warrior-dev"/>
+  <project name="armmbed/meta-mbl" path="layers/meta-mbl" remote="github"/>
   <project name="git/meta-freescale" path="layers/meta-freescale" remote="yocto"/>
+  <project name="git/meta-security" path="layers/meta-security" remote="yocto"/>
   <project name="git/poky" path="layers/poky" remote="yocto"/>
+  <project name="openembedded/meta-openembedded" path="layers/meta-openembedded" remote="github"/>
+  <project name="openembedded/openembedded-core" path="layers/openembedded-core" remote="github"/>
```

Comment:
Doesn't the poky repo (that we're putting in layers/poky) already have a copy of oe-core?

Reply:
Yes, it does. I'll raise a ticket to address this duplicity.

Reply:
I don't really like leaving it like this, but it's not really a problem with the zeus-dev->master merge, it's a problem with the code in zeus-dev that's being merged, so, okay.

Reply:
(Will approve when there's a reference to the new ticket).

Reply:
https://jira.arm.com/browse/IOTMBL-2675